disable using git because it should not be used nor needed
Written and tested by Lucas Mitrak.

--- a/PokerTH_tracker.pro
+++ b/PokerTH_tracker.pro
@@ -46,10 +46,10 @@ TRANSLATIONS = Lang/PokerTH_tracker_de.ts
 
 system("lrelease PokerTH_tracker.pro")
 
-GIT_COMMIT = $$system("git log -1 --format=%H")
-system("sed -e \"s/@GIT_COMMIT@/$$GIT_COMMIT/g\" misc/version.hpp.in > version.hpp")
+#GIT_COMMIT = $$system("git log -1 --format=%H")
+#system("sed -e \"s/@GIT_COMMIT@/$$GIT_COMMIT/g\" misc/version.hpp.in > version.hpp")
 
 desktop.path = /usr/share/applications
 desktop.files += PokerTH_tracker.desktop
